ProcessLib/GroundwaterFlowFEM.h

```diff
@@ -80,7 +81,10 @@ class LocalAssemblerData : public LocalAssemblerDataInterface<GlobalMatrix, Glob
                     _shape_matrices[ip]);
         }
 
-        _hydraulic_conductivity = hydraulic_conductivity;
+        _hydraulic_conductivity = [&hydraulic_conductivity, &e]()
```

Comment:
With time (or other values for non-linear equations) dependency this will become a function anyway. So this is more conceptual thing here, about which I'm still not very happy.

If for a particular process a constant is needed per element, saving a constant here will save few bytes because the std::function<...> is relatively heavy object.